### ISSUE TITLE
error: exception in "temp = realloc(new->read_buf, new->read_size * sizeof(_Complex float))" 

### ISSUE THREAD

**Issue by Marcotic** [BatchDrake/SigDigger#124]
Got this error and libusb: error [udev_hotplug_event] ignoring udev action bind on console while trying to capture with hackrf

**Comment by BatchDrake** [comment]
I don't think that message is related to this problem. Instead, it's like the device is reporting a 0-sized MTU, which is plainly wrong. What SoapySDR version are you using and in which operating system? Also, are you building from sources or using the AppImages?

**Comment by Marcotic** [comment]
SoapySDRUtil --info:

Lib Version: v0.8.0-ga8df1c57
API Version: v0.8.0
ABI Version: v0.8

On debian 10, building sources...
